Test spectra unittests failing

Unittests give the error

```
======================================================================
ERROR: test_slicing (test_spectra.TestSpectra)
Test the slicing shirnks the spectra in the correct way.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ashley/snoplus/software/echidna/echidna/echidna/test/test_spectra.py", line 94, in test_slicing
    test_spectra._time_high))
  File "echidna/core/spectra.py", line 154, in shrink
    raise ValueError("Energy high is below exist bound")
ValueError: Energy high is below exist bound
```

The `assertRaises` method takes the function and the function's separate arguments as arguments, e.g.:

```
self.assertRaises(ValueError,
                  test_spectra.shrink,
                  test_spectra._energy_low,
                  2 * test_spectra._energy_high,
                  test_spectra._radial_low,
                  test_spectra._radial_high,
                  test_spectra._time_low,
                  test_spectra._time_high)
```

But I then get

```
======================================================================
FAIL: test_slicing (test_spectra.TestSpectra)
Test the slicing shirnks the spectra in the correct way.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ashley/snoplus/software/echidna/echidna/echidna/test/test_spectra.py", line 120, in test_slicing
    test_spectra._time_bins / 2))
AssertionError: False is not true

----------------------------------------------------------------------
```

Which I don't know how to fix

--- a/echidna/core/spectra.py
+++ b/echidna/core/spectra.py
@@ -149,17 +149,17 @@ class Spectra(object):
         existing binning low). Finally update all the bookeeping and slice.
         """
         if(energy_low is not None and energy_low < self._energy_low):
-            raise ValueError("Energy low is below exist bound")
+            raise ValueError("Energy low is below existing bound")
         if(energy_high is not None and energy_high > self._energy_high):
-            raise ValueError("Energy high is below exist bound")
+            raise ValueError("Energy high is above existing bound")
         if(radial_low is not None and radial_low < self._radial_low):
-            raise ValueError("Radial low is below exist bound")
+            raise ValueError("Radial low is below existing bound")
         if(radial_high is not None and radial_high > self._radial_high):
-            raise ValueError("Radial high is below exist bound")
+            raise ValueError("Radial high is above existing bound")
         if(time_low is not None and time_low < self._time_low):
-            raise ValueError("Time low is below exist bound")
+            raise ValueError("Time low is below existing bound")
         if(time_high is not None and time_high > self._time_high):
-            raise ValueError("Time high is below exist bound")
+            raise ValueError("Time high is above existing bound")
 
         energy_low_bin = 0
         energy_high_bin = self._energy_bins

--- a/echidna/test/test_spectra.py
+++ b/echidna/test/test_spectra.py
@@ -86,32 +86,35 @@ class TestSpectra(unittest.TestCase):
         test_decays = 10
         test_spectra = spectra.Spectra("Test", test_decays)
         self.assertRaises(ValueError,
-                          test_spectra.shrink(test_spectra._energy_low,
-                                              2 * test_spectra._energy_high,
-                                              test_spectra._radial_low,
-                                              test_spectra._radial_high,
-                                              test_spectra._time_low,
-                                              test_spectra._time_high))
+                          test_spectra.shrink,
+                          test_spectra._energy_low,
+                          2 * test_spectra._energy_high,
+                          test_spectra._radial_low,
+                          test_spectra._radial_high,
+                          test_spectra._time_low,
+                          test_spectra._time_high)
         self.assertRaises(ValueError,
-                          test_spectra.shrink(test_spectra._energy_low,
-                                              test_spectra._energy_high,
-                                              test_spectra._radial_low,
-                                              2 * test_spectra._radial_high,
-                                              test_spectra._time_low,
-                                              test_spectra._time_high))
+                          test_spectra.shrink,
+                          test_spectra._energy_low,
+                          test_spectra._energy_high,
+                          test_spectra._radial_low,
+                          2 * test_spectra._radial_high,
+                          test_spectra._time_low,
+                          test_spectra._time_high)
         self.assertRaises(ValueError,
-                          test_spectra.shrink(test_spectra._energy_low,
-                                              test_spectra._energy_high,
-                                              test_spectra._radial_low,
-                                              test_spectra._radial_high,
-                                              test_spectra._time_low,
-                                              2 * test_spectra._time_high))
-        test_spectra.shrink(2 * test_spectra._energy_low,
-                            test_spectra._energy_high,
-                            2 * test_spectra._radial_low,
-                            test_spectra._radial_high,
-                            2 * test_spectra._time_low,
-                            test_spectra._time_high)
-        self.assertTrue(test_spectra._data.shape == (test_spectra._energy_bins / 2,
-                                                     test_spectra._radial_bins / 2,
-                                                     test_spectra._time_bins / 2))
+                          test_spectra.shrink,
+                          test_spectra._energy_low,
+                          test_spectra._energy_high,
+                          test_spectra._radial_low,
+                          test_spectra._radial_high,
+                          test_spectra._time_low,
+                          2 * test_spectra._time_high)
+        test_spectra.shrink(test_spectra._energy_low,
+                            test_spectra._energy_high / 2,
+                            test_spectra._radial_low,
+                            test_spectra._radial_high / 2,
+                            test_spectra._time_low,
+                            test_spectra._time_high / 2)
+        self.assertTrue(test_spectra._data.shape == (test_spectra._energy_bins,
+                                                     test_spectra._radial_bins,
+                                                     test_spectra._time_bins))

